build-tests/api-extractor-test-01/dist/public/api-extractor-test-01.d.ts

```diff
@@ -1,194 +0,0 @@
-/**
```

Comment:
Don't delete these.  This is what we're testing -- basically I'm looking for diffs in this file when I make changes to the *.d.ts generator (even though it's not validated by the build)